### ISSUE TITLE
Support for default routes with `route` resource

### ISSUE THREAD

**Issue by spiffytech** [chef/chef#2243]
[This bug](https://tickets.opscode.com/browse/CHEF-3176) appears to have not made it over from the old bug tracker.

The below compiled resource should be equivalent to the command
`ip route replace default via 192.168.20.5`
and produce an entry from the `ip route` command that looks like this:
`default via 192.168.20.5 dev eth0`
When trying to set the default route on my system, I get an error (ArgumentError for "invalid address").

Compiled Resource: 

```
route("default") do 
  action [:add]
  retries 0 
  retry_delay 2 
  target "default" 
  gateway "192.168.20.5" 
  device "eth0" 
  route_type :host 
  cookbook_name :webserver_onsite 
  recipe_name "routes" 
end
```

**Comment by anastazya** [comment]
Workaround for this bug , at least for me : 

execute "route_add" do
  command "route add default gw 192.168.88.1 eth0"
  action :run
  not_if "ip route show | grep default"
end

I hope this helps someone ..

**Comment by gairadzi** [comment]
using '0.0.0.0/0' instead of 'default' also solves the issue

**Comment by tas50** [comment]
This fix shipped in Chef 13.4.24
